sdk/lib/atl/atlbase.h

```diff
@@ -1856,6 +1859,61 @@ inline HRESULT WINAPI AtlComModuleRevokeClassObjects(_ATL_COM_MODULE *module)
     return S_OK;
 }
 
+#if DBG
```

Comment:
ATL Tracing functionality is defined in `atltrace.h`

Reply:
OK. Moved.